main.go

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
I'd move this file to the cmd package, as we do not want to interfere with the `testcontainers` package of the library